src/components/LoadingComment/index.js

```diff
@@ -46,7 +46,7 @@ const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.blackgrey` to `COLORS.gray400` for the `color` property in the `time` style is again a simple color change and should not affect the functionality of the component.
```diff
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->